Gradle `runKsp` task fails to generate MCP schemas

Gradle `runKsp` tasks of feature modules fail with `Task with path 'kspKotlinDesktop' not found` error.

```
$ ./scripts/generate_mcp_schemas.sh
Generating MCP schema files...
Target directory: ../composeflow-api-ts/schemas
Running KSP to generate JSON schema files...
INFO: Copied /Users/tkanda/src/github.com/ComposeFlow/ComposeFlow/gradle/libs.versions.toml to /Users/tkanda/src/github.com/ComposeFlow/ComposeFlow/feature/app-builder/app-template/gradle/libs.versions.toml during initialization.

> Configure project :build-logic:plugins
WARNING: Unsupported Kotlin plugin version.
The `embedded-kotlin` and `kotlin-dsl` plugins rely on features of Kotlin `2.0.20` that might work differently than in the requested version `2.2.0`.

> Configure project :desktopApp
w: ⚠️ 'org.jetbrains.kotlin.multiplatform' plugin 'withJava()' configuration deprecation.
Kotlin multiplatform plugin always configures Java sources compilation and 'withJava()' configuration is deprecated.
Solution:
Please remove 'withJava()' method call from build configuration.

[Incubating] Problems report is available at: file:///Users/tkanda/src/github.com/ComposeFlow/ComposeFlow/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':feature:uibuilder:runKsp'.
> Task with path 'kspKotlinDesktop' not found in project ':feature:uibuilder'.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/8.11.1/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 976ms
4 actionable tasks: 4 up-to-date
INFO: Copied /Users/tkanda/src/github.com/ComposeFlow/ComposeFlow/gradle/libs.versions.toml to /Users/tkanda/src/github.com/ComposeFlow/ComposeFlow/feature/app-builder/app-template/gradle/libs.versions.toml during initialization.

> Configure project :build-logic:plugins
WARNING: Unsupported Kotlin plugin version.
The `embedded-kotlin` and `kotlin-dsl` plugins rely on features of Kotlin `2.0.20` that might work differently than in the requested version `2.2.0`.

> Configure project :desktopApp
w: ⚠️ 'org.jetbrains.kotlin.multiplatform' plugin 'withJava()' configuration deprecation.
Kotlin multiplatform plugin always configures Java sources compilation and 'withJava()' configuration is deprecated.
Solution:
Please remove 'withJava()' method call from build configuration.

[Incubating] Problems report is available at: file:///Users/tkanda/src/github.com/ComposeFlow/ComposeFlow/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':feature:appstate-editor:runKsp'.
> Task with path 'kspKotlinDesktop' not found in project ':feature:appstate-editor'.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/8.11.1/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 725ms
4 actionable tasks: 4 up-to-date
INFO: Copied /Users/tkanda/src/github.com/ComposeFlow/ComposeFlow/gradle/libs.versions.toml to /Users/tkanda/src/github.com/ComposeFlow/ComposeFlow/feature/app-builder/app-template/gradle/libs.versions.toml during initialization.

> Configure project :build-logic:plugins
WARNING: Unsupported Kotlin plugin version.
The `embedded-kotlin` and `kotlin-dsl` plugins rely on features of Kotlin `2.0.20` that might work differently than in the requested version `2.2.0`.

> Configure project :desktopApp
w: ⚠️ 'org.jetbrains.kotlin.multiplatform' plugin 'withJava()' configuration deprecation.
Kotlin multiplatform plugin always configures Java sources compilation and 'withJava()' configuration is deprecated.
Solution:
Please remove 'withJava()' method call from build configuration.

[Incubating] Problems report is available at: file:///Users/tkanda/src/github.com/ComposeFlow/ComposeFlow/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':feature:datatype-editor:runKsp'.
> Task with path 'kspKotlinDesktop' not found in project ':feature:datatype-editor'.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/8.11.1/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 717ms
4 actionable tasks: 4 up-to-date
Error: Failed to run KSP. Exiting.
```

--- a/feature/appstate-editor/build.gradle.kts
+++ b/feature/appstate-editor/build.gradle.kts
@@ -63,8 +63,8 @@ tasks.register("runKsp") {
         mkdir("${project.buildDir}/generated/llm-tools")
     }
 
-    // Depend on the KSP task for the desktop target
-    dependsOn("kspKotlinDesktop")
+    // Depend on the KSP task for the JVM target
+    dependsOn("kspKotlinJvm")
 }
 
 // Make sure the KSP tasks run

--- a/feature/datatype-editor/build.gradle.kts
+++ b/feature/datatype-editor/build.gradle.kts
@@ -66,8 +66,8 @@ tasks.register("runKsp") {
         mkdir("${project.buildDir}/generated/llm-tools")
     }
 
-    // Depend on the KSP task for the desktop target
-    dependsOn("kspKotlinDesktop")
+    // Depend on the KSP task for the JVM target
+    dependsOn("kspKotlinJvm")
 }
 
 // Make sure the KSP tasks run

--- a/feature/string-editor/build.gradle.kts
+++ b/feature/string-editor/build.gradle.kts
@@ -66,7 +66,7 @@ tasks.register("runKsp") {
         mkdir("${project.buildDir}/generated/llm-tools")
     }
 
-    // Depend on the KSP task for the desktop target
+    // Depend on the KSP task for the JVM target
     dependsOn("kspKotlinJvm")
 }
 

--- a/feature/string-editor/build.gradle.kts
+++ b/feature/string-editor/build.gradle.kts
@@ -67,7 +67,7 @@ tasks.register("runKsp") {
     }
 
     // Depend on the KSP task for the desktop target
-    dependsOn("kspKotlinDesktop")
+    dependsOn("kspKotlinJvm")
 }
 
 // Make sure the KSP tasks run

--- a/feature/uibuilder/build.gradle.kts
+++ b/feature/uibuilder/build.gradle.kts
@@ -93,8 +93,8 @@ tasks.register("runKsp") {
         mkdir("${project.buildDir}/generated/llm-tools")
     }
 
-    // Depend on the KSP task for the desktop target
-    dependsOn("kspKotlinDesktop")
+    // Depend on the KSP task for the JVM target
+    dependsOn("kspKotlinJvm")
 }
 
 // Make sure the KSP tasks run

